sass/_navigation.scss

```diff
@@ -5,223 +5,226 @@
     }
 
-	&:hover,
-    &:focus,
-    &:active {
-        color: $link-color-active;
-    }
-
-	&:hover {
-        color: $link-color-active;
-		text-decoration: none;
+    &:hover {
+      border-color: transparent;
 
-		.entry-summary &,
-        .entry-content &,
-        .edit-link &,
-        .entry-meta & {
-            border-color: $link-border-color;
-        }
+      .name {
+        border-color: $link-border-color;
+      }
     }
-
-	&.author {
-		.name {
-			border-bottom: 1px solid transparent;
-			transition: border .2s ease-in-out;
-		}
-
-		&:hover {
-			border-color: transparent;
-
-			.name {
-				border-color: $link-border-color;
-			}
-		}
-	}
+  }
 }
 
 /*--------------------------------------------------------------
 5.2 Menus
 --------------------------------------------------------------*/
 .main-navigation {
-	clear: both;
-	display: block;
-	float: left;
-	width: 100%;
-
-	ul {
-        list-style: none;
-        margin: 0;
-        padding-left: 0;
-
-        &.hide {
-            display: none;
-        }
-    }
-
-    a {
-        border-bottom: 1px solid #ddd;
-        display: block;
-        padding: 10px 0;
-        text-decoration: none;
+  clear: both;
+  display: block;
+  float: left;
+  width: 100%;
+
+  ul {
+    list-style: none;
+    margin: 0;
+    padding-left: 0;
+
+    &.hide {
+      display: none;
     }
+  }
+
+  a {
+    border-bottom: 1px solid #ddd;
+    display: block;
+    padding: 10px 0;
+    text-decoration: none;
+  }
 }
 
 .site-main .comment-navigation,
 .site-main .paging-navigation {
-	margin: 0 0 1.5em;
-	overflow: hidden;
-    padding: 50px 0;
-    @include site-width($width-medium);
+  margin: 0 0 1.5em;
+  overflow: hidden;
+  padding: 50px 0;
+  @include site-width($width-medium);
 }
 
 .paging-navigation a {
-    @include button($color-blue);
-
-    @media screen and (max-width: $width-phone) {
-        display: block;
-        margin-bottom: 10px;
-        text-align: center;
-        width: auto;
-    }
+  @include button($color-blue);
+
+  @media screen and (max-width: $width-phone) {
+    display: block;
+    margin-bottom: 10px;
+    text-align: center;
+    width: auto;
+  }
 }
 
 .comment-navigation .nav-previous,
 .paging-navigation .nav-previous {
-	float: left;
+  float: left;
 }
 
 .comment-navigation .nav-next,
 .paging-navigation .nav-next {
-	float: right;
-	text-align: right;
+  float: right;
+  text-align: right;
 }
 
 @media screen and (max-width: $width-phone) {
-	.comment-navigation .nav-previous,
-	.paging-navigation .nav-previous,
-	.comment-navigation .nav-next,
-	.paging-navigation .nav-next {
-		float: none;
-		width: auto;
-	}
+  .comment-navigation .nav-previous,
+  .paging-navigation .nav-previous,
+  .comment-navigation .nav-next,
+  .paging-navigation .nav-next {
+    float: none;
+    width: auto;
+  }
 }
 
 .post-navigation {
-    background: #eee;
-    margin-top: 50px;
-
-    [class$="title"] {
-        margin: 0 0 30px;
+  background: #eee;
```

Comment:
Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.